telemetry/specs/INT.mdk

```diff
@@ -1088,28 +1106,25 @@ header.
     a version 1.0 INT switch cannot operate alongside version 2.0 INT switches
     if a new 8B metadata is introduced in version 2.0, as the version 1.0
     INT switch could insert 0xFFFFFFFF reserved value for a 8B metadata field,
-    thus breaking the metadata stack length invariance - the length of
-    metadata stack will not be a multiple of Per-Hop Metadata length \* 4
-    in this case.
+    thus breaking the metadata length invariance - the length of
+    metadata stack (excluding 'source-only' Domain Specific Metadata)
+    must be a multiple of Hop ML \* 4.
```

Comment:
I think most of the text above is no longer relevant. Instead, you have to follow the rules at 1059 above. It does not matter whether the unknown bits are from the base instruction bitmap or the DS instructions, either way you have to pad out to Hop ML.

Reply:
Removed lines 1109-1111. 
Kept the text about the major revision requirement for adding non-4B metadata. The padding out to Hop ML must honor the order and the size of each metadata.